src/schema/feeds.ts

```diff
@@ -230,6 +230,46 @@ export const typeDefs = /* GraphQL */ `
       supportedTypes: [String!]
     ): PostConnection! @auth
 
+    """
+    Get feed preview
+    """
+    feedPreview(
+      """
+      Time the pagination started to ignore new items
+      """
+      now: DateTime
+
+      """
+      Paginate after opaque cursor
+      """
+      after: String
+
+      """
+      Paginate first
+      """
+      first: Int
+
+      """
+      Ranking criteria for the feed
+      """
+      ranking: Ranking = POPULARITY
+
+      """
+      Return only unread posts
+      """
+      unreadOnly: Boolean = false
+
+      """
+      Version of the feed algorithm
+      """
+      version: Int = 1
+
+      """
+      Array of supported post types
+      """
+      supportedTypes: [String!]
+    ): PostConnection! @auth
```

Comment:
I added all the default args that other feeds have to support in future. Was thinking we might want to not allow some of them for preview, WDYT?